app/javascript/util/listingUtil.ts

```diff
@@ -72,3 +73,53 @@ export const getListingAddressString = (listing: RailsListing): string => {
     ""
   )
 }
+
+/**
+ * Check if a listing is accepting paper applications
+ * @param {RailsRentalListing | RailsRentalListing} listing
+ * @returns {boolean} returns true if the listing is accepting paper applications, false otherwise
+ */
+export const acceptingPaperApplications = (listing: RailsListing): boolean => {
+  return listing.Accepting_applications_at_leasing_agent || listing.Accepting_applications_by_PO_Box
+}
+
+export const mohcdPaperAppURLBase = "https://sfmohcd.org/sites/default/files/Documents/MOH/"
+
+export const mohcdRentalPaperAppURLTemplate =
+  mohcdPaperAppURLBase +
+  "BMR%20Rental%20Paper%20Applications/" +
+  "{lang}%20BMR%20Rent%20Short%20Form%20Paper%20App.pdf"
+
+export const mohcdSalePaperAppURLTemplate =
+  mohcdPaperAppURLBase +
+  "BMR%20Ownership%20Paper%20Applications/" +
+  "{lang}%20BMR%20Own%20Short%20Form%20Paper%20App.pdf"
+
+export const paperAppLanguages = [
+  { language: "English", prefix: "en" },
+  { language: "Spanish", prefix: "es" },
+  { language: "Chinese", prefix: "zh" },
+  { language: "Tagalog", prefix: "tl" },
+]
+
+type PaperApplication = {
+  languageString: string
+  fileURL: string
+}
+
+/**
+ * Returns an array of all the paper application URLs
+ * @param {boolean} isRental
+ * @returns {PaperApplication[]}
+ */
+export const paperApplicationURLs = (isRental: boolean): PaperApplication[] => {
+  const urlBase = isRental ? mohcdRentalPaperAppURLTemplate : mohcdSalePaperAppURLTemplate
+  return paperAppLanguages.map(
+    (lang): PaperApplication => {
+      return {
+        languageString: LANGUAGE_CONFIGS[lang.prefix].getLabel(),
+        fileURL: urlBase.replace("{lang}", lang.language),
+      }
```

Comment:
unit test for this function would be cool to make sure we're building all the url's properly